practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/client/AbstractEntityServiceClient.java

```diff
@@ -11,13 +12,17 @@
     protected WebClient webClient;
     protected String serviceUrlBase;
     protected String serviceExistsRoute;
+    protected SecurityContextUtil securityContextUtil;
+
+
 
     @SuppressWarnings("ConstantConditions")
     @Override
     public boolean entityIdExists(String id) throws RestException {
         Objects.requireNonNull(id, "id must be non-null");
         return webClient.get()
                 .uri(serviceExistsRoute, id)
+                .header("Authorization", "Bearer " + securityContextUtil.getToken())
```

Comment:
I think it is better to use the constant:
`.header(HttpHeaders.AUTHORIZATION, "Bearer " + token)`

Same in other calls...

Reply:
Thanks Liza - good point.   I have changed this here in the practitioner service and also in security/src/main/java/uk/ac/ox/ndph/mts/client/site_service/SiteServiceClientImpl.java

Reply:
If you use the headers() method you can use:
`.uri(...route..).headers( (headers) -> headers.setBearerAuth(token) )`
- or pass a `Consumer<Headers>` customizer - see role-service-client